html/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>myhomepage</title>
```

Comment:
```suggestion
    <title>Tobias Schweizer</title>
```
This will be visible on the Browser Tab, so it's nice to get a specific Idea of the content :)